doc/ref/spec/opcodes.md

```diff
@@ -0,0 +1,46 @@
+6   |pcall        |Takes two words by which it is followed. They are address $a$ and the number of parameters $n$, respectively. Then it executes the smart contract with the address $a$ and passes there only $n$ top elements of the stack.                                                                 
+7   |lcall        |Takes three words by which it is followed.They are address $a$, function $f$ and the number of parameters $n$, respectively. Then it executes the function $f$ of the library (which is a special form of program) with the address $a$ and passes there only $n$ top elements of the stack.
+10  |pop          |Removes first item from the stack.                                                                                                                                                                                                                                                          
+11  |push         |Pushes the word following the opcode to the stack.                                                                                                                                                                                                                                          
+12  |dup          |Duplicates first item of the stack.                                                                                                                                                                                                                                                         
+13  |dupn         |Duplicates `(n+1)`-th item of the stack where `n` is the first item in stack.                                                                                                                                                                                                               
+14  |swap         |Swaps first two items in the stack.                                                                                                                                                                                                                                                         
+15  |swapn        |Swaps the second item of the stack with the `(n+1)`-th item of the stack where `n` is first item in the stack.                                                                                                                                                                              
+20  |new          |Puts the data following the opcode to the heap. Pushes reference to the stack.                                                                                                                                                                                                              
```

Comment:
It's better to write `0x<code>`, it might look confusing when you read just `20`